index.html

```diff
@@ -352,40 +352,40 @@ <h3>
         };
         </pre>
         <p data-dfn-for='PerformanceNavigationTiming'>
-          If the previous document, and any HTTP redirects or equivalent when
-          navigating, pass the <a href=
-          "https://www.w3.org/TR/resource-timing-2/#timing-allow-check">timing
+          If the previous document, and any <a data-cite="FETCH#redirect-status">HTTP
+          redirects</a> when navigating, pass the <a href=
+          "https://www.w3.org/TR/resource-timing-2/#dfn-timing-allow-check">timing
           allow check</a> algorithm, the <dfn>unloadEventStart</dfn> attribute
           MUST return a <a href=
           "https://www.w3.org/TR/hr-time-2/#dom-domhighrestimestamp">DOMHighResTimeStamp</a>
           with a time value equal to the time immediately before the user agent
           starts the <a data-cite=
-          "!HTML/browsing-the-web.html#unloading-documents">unload event</a> of the
+          "HTML/browsing-the-web.html#unloading-documents">unload event</a> of the
           previous document.
         </p>
         <p>
           If there is no previous document or the above <a href=
-          "https://www.w3.org/TR/resource-timing-2/#timing-allow-check">timing
+          "https://www.w3.org/TR/resource-timing-2/#dfn-timing-allow-check">timing
```

Comment:
as above... you can remove all `href=""`s

Reply:
Note: if you define the following somewhere (e.g., a Dependencies section):
```
<dfn data-cite="resource-timing-2#dfn-timing-allow-check">timing allow check</dfn>
```
then you can just do this everywhere:

```HTML
<a>timing allow check</a>
```

And it will automatically just link to the right spec/place.